osu.Game/Overlays/Settings/Sections/Input/TabletAreaSelection.cs

```diff
@@ -212,17 +212,17 @@ protected override void Update()
         {
             base.Update();
 
-            if (!(tablet.Value?.Size is Vector2 size))
-                return;
```

Comment:
I prefer the old version of the code here. We generally prefer early-returns to nested `if`s.